### PR TITLE
Cleanup and switch Freya.Console to Suave

### PR DESCRIPTION
**Pull request by panesofglass**
I've added this directly into the main Freya repo to make it easier to debug.

cc: @haf @kolektiv

### CODE REVIEW

File: tests/Freya.Console/Program.fs [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
+          Encodings = None
+          MediaType = Some MediaType.Text
+          Languages = Some [ LanguageTag.Parse "en" ] }
+      Data = Encoding.UTF8.GetBytes x }
+
 let ok _ =
-        Freya.Lens.setPartial Response.ReasonPhrase_ "Hey Folks!"
-     *> Freya.init Representation.empty
+        Freya.Lens.setPartial Response.ReasonPhrase_ "Hey, folks!"
+     *> Freya.init (represent "Hey, folks!")
```
Comment (panesofglass) [inline tests/Freya.Console/Program.fs]: This should now attempt to write to the body.

**Comment by panesofglass** [comment]
Suave issues with Freya.Machine appear to be Freya related. See https://github.com/SuaveIO/suave/pull/350

File: src/Freya.Machine.Extensions.Http/Handlers.fs [hunk 2]
```diff
@@ -96,7 +96,9 @@ let private description x =
      *> languages x.Languages
 
 let private data x =
-    Response.body_ %= (fun body -> body.Write (x, 0, x.Length); body)
+    freya {
+        let! body = Freya.Optic.get Response.body_
+        body.Write (x, 0, x.Length) }
```
Comment (panesofglass) [inline src/Freya.Machine.Extensions.Http/Handlers.fs]: @kolektiv my attempt to fix machine writing for Suave. This does not appear to have worked, but I committed it b/c I understand this far better than the previous one-liner. Feel free to revert.

**Comment by kolektiv** [comment]
See https://github.com/SuaveIO/suave/issues/357 ...

**Pull request closed without merging**